db.py

```diff
@@ -113,6 +113,27 @@ class Vote(Base):
                        primary_key=True)
     user = Column(String(8), primary_key=True)
 
+
+class Playlist(Base):
+    __tablename__ = 'playlists'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(100))
+    user = Column(String(8))
+
+
+class PlaylistItem(Base):
+    __tablename__ = 'playlist_items'
+
+    playlist_id = Column(
+        Integer, ForeignKey('playlists.id', ondelete='CASCADE'),
+        primary_key=True)
+    index = Column(Integer, primary_key=True, autoincrement=False)
+    song_id = Column(Integer, ForeignKey('songs.id', ondelete='CASCADE'))
+
+    __table_args__ = (UniqueConstraint('playlist_id', 'song_id'),)
```

Comment:
It was that way when I started expanding the functionality, but I think it could be changed so that songs are removed by index and not song_id.